app/celer-dump-data.cc

```diff
@@ -131,53 +131,67 @@ void print_materials(std::vector<ImportMaterial>& materials,
     //// PRINT CUTOFF LIST ///
 
     cout << R"gfm(
-## Cutoffs per material
+## Cutoffs
 
-| Material ID | Name                            | Cutoffs [MeV, cm]               |
-| ----------- | ------------------------------- | ------------------------------- |
+| Material                   | Particle  | Energy [MeV] | Range [cm] |
+| -------------------------- | --------- | ------------ | ---------- |
 )gfm";
 
-    std::map<int, std::string> pdg_label;
-    for (auto particle_id : range(ParticleId{particles.size()}))
-    {
-        int const pdg = particles.id_to_pdg(particle_id).get();
-        auto const& label = particles.id_to_label(particle_id);
-        pdg_label.insert({pdg, label});
-    }
-
     for (unsigned int material_id : range(materials.size()))
     {
         bool is_first_line = true;
         auto const& material = materials[material_id];
 
-        for (auto const& cutoff_key : material.pdg_cutoffs)
+        for (auto const& [pdg, cuts] : material.pdg_cutoffs)
```

Comment:
This is a C++ 17 feature. Are we fine with a warning?

Reply:
Yep :) now we can use all the C++17 😈